src/drive/components/FileList.jsx

```diff
@@ -97,20 +42,21 @@ class FileList extends PureComponent {
       onFileToggle,
       showActionMenu,
       withSelectionCheckbox,
-      withFilePath
+      withFilePath,
+      isRenaming,
+      renamingFile,
+      isAvailableOffline
     } = this.props
     const file = this.props.files[index]
     if (!file) {
-      return <FilePlaceholder key={key} style={style} />
+      return <FilePlaceholder key={key} />
```

Comment:
Hmm are you sure this is still necessary? `<FilePlaceholder />` was used to display something when some files were loaded by react-virtualized, but I can't see how `file` could be evaluated to false here.

Reply:
À partir du moment ou on récupère l'index et non le fichier (besoin d'un peu de refacto), on passe l'instruction `const file = this.props.files[index]` et donc avoir un `file` falsy...
On le virera quand on passera directement la propriété `file` au composant `File`, non ?